docs/source/changelog.rst

```diff
@@ -1,6 +1,28 @@
 Changelog
 =========
 
+2024.3.0
+--------
+
+Enhancements
+
+- coroutines throttle to stream pool rather than batches (#1544)
+- write transactions in simplecache (#1531)
+- allow deep nested refs in referenceFS/parquet (#1530)
+
+Fixes
+
+- remove extra calling mapper contains (#1546)
```

Comment:
You forgot #1536 , which is a pretty big behavior change.

Reply:
Please make a new PR for it?